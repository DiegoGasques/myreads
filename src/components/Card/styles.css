.card {
  position: relative;
  max-width: 100%;
  min-width: 140px;
  display: flex;
  flex-flow: row;
  /* background-color: #ecf0f1; */
  border-radius: 0.25rem;
  margin: 1rem;
  user-select: none;
  transition: 0.25s;
}
