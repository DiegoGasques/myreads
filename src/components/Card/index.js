import React from "react";
import classnames from "classnames";
import PropTypes from "prop-types";

import "./styles.css";

function Card({ className, children }) {
  return <div className={classnames("card", className)}>{children}</div>;
}

Card.propTypes = {
  className: PropTypes.string,
  children: PropTypes.node
};

export default Card;
