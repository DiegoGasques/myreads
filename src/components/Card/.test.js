import React from "react";
import Card from "./index";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

it("Should match snapshot", () => {
  const wrapper = shallow(
    <Card>
      <h1>Test</h1>
    </Card>
  );
  expect(toJSON(wrapper)).toMatchSnapshot();
});
