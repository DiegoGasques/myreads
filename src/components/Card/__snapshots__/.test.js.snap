// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div
  className="card"
>
  <h1>
    Test
  </h1>
</div>
`;
